javascript-light-bulb/main.js

```diff
@@ -0,0 +1,13 @@
+var $lightbulb = document.querySelector('.circle-night');
+var $room = document.querySelector('.background-night');
+function clicked() {
+  if ($lightbulb.className === 'circle-night') {
```

Comment:
You shouldn't base your functionality on the current classes applied to an element. What if another piece of code adjusted the class list somehow? Instead, you should have a variable that holds a value to let you determine whether you should turn the light on or off whenever the button is clicked.